pages/en/network/benefits.mdx

```diff
@@ -4,25 +4,27 @@ navTitle: Benefits
 socialImage: https://thegraph.com/docs/img/seo/benefits.jpg
 ---
 
-The Graph’s decentralized network has been engineered and refined to create a robust indexing and querying experience—and it’s getting better every day thanks to thousands of contributors around the world. The benefits of this decentralized protocol cannot be replicated by running a graph node locally. The Graph Network is more reliable, more efficient, and less expensive. Here is an analysis:
+The Graph’s decentralized network has been engineered and refined to create a robust indexing and querying experience—and it’s getting better every day thanks to thousands of contributors around the world.
+
+The benefits of this decentralized protocol cannot be replicated by running a graph node locally. The Graph Network is more reliable, more efficient, and less expensive.
```

Comment:
Should "graph node" be all lowercase? Maybe we should say "running `graph-node"` instead?

Reply:
Good call - `graph-node` would look a lot better.